frontend/src/App.tsx

```diff
@@ -9,27 +9,31 @@ import Registration from "./pages/Registration/Registration";
 import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
 import Login from "./pages/Login/Login";
-import OwnDrink from "./pages/OwnDrink/OwnDrink";
+import CustomDrink from "./pages/CustomDrink/CustomDrink";
+
 
 
 export default function App() {
 
 
+
+
     return (
 
     <>            <div className={"App"}>
         <Header />
         <div  className={"main-part"}>
+
         <Routes>
 
             <Route path={"/"} element={<Login />}/>
             <Route path={"/register"} element={<Registration />}/>
             <Route path={"/home"} element={<Home />} />
             <Route path={"/ingredient"} element={<Ingredient />} />
             <Route path={"/favourites"} element={<Favourite />}/>
-            <Route path={"/ownDrink"} element={<OwnDrink />} />
+            <Route path={"/customDrink"} element={<CustomDrink />} />
             <Route path={"/ingredient=:drinkCategory"} element={<Category />} />
-            <Route path={"/details=:details"} element={<Details  />} />
+            <Route path={"/details=:details&source=:source"} element={<Details />} />
```

Comment:
Is there a reason, why you make your url's this long? Wouldn't : 
 ```suggestion
            <Route path={"/details/:details/:source"} element={<Details />} />
```
be just as fine and also more readable?